querydsl-sql/src/test/java/com/querydsl/sql/SubqueriesBase.java

```diff
@@ -15,7 +29,11 @@
 import com.google.common.collect.ImmutableList;
 import com.querydsl.core.testutil.ExcludeIn;
 import com.querydsl.core.types.SubQueryExpression;
-import com.querydsl.core.types.dsl.*;
+import com.querydsl.core.types.dsl.Expressions;
+import com.querydsl.core.types.dsl.NumberPath;
+import com.querydsl.core.types.dsl.Param;
+import com.querydsl.core.types.dsl.PathBuilder;
```

Comment:
You could open a new pull request, and link it from here.
At least, if the rest of the team thinks it's a good idea.